catalog/spatial/kml/spatial-kml-util/src/main/java/org/codice/ddf/spatial/kml/converter/MetacardToKml.java

```diff
@@ -0,0 +1,142 @@
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.kml.converter;
+
+import com.vividsolutions.jts.geom.GeometryCollection;
+import com.vividsolutions.jts.geom.LineString;
+import com.vividsolutions.jts.geom.Point;
+import com.vividsolutions.jts.geom.Polygon;
+import com.vividsolutions.jts.io.ParseException;
+import com.vividsolutions.jts.io.WKTReader;
+import ddf.catalog.transform.CatalogTransformerException;
+import de.micromata.opengis.kml.v_2_2_0.Coordinate;
+import de.micromata.opengis.kml.v_2_2_0.Geometry;
+import de.micromata.opengis.kml.v_2_2_0.KmlFactory;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+
+public class MetacardToKml {
+
+  private static final String POINT_TYPE = "Point";
+
+  private static final String LINES_STRING_TYPE = "LineString";
+
+  private static final String POLYGON_TYPE = "Polygon";
+
+  /**
+   * Convert wkt string into JTS Geometry and then from JTS to KML Geo.
+   *
+   * @param wkt
+   * @return
+   * @throws CatalogTransformerException
+   */
+  public static Geometry getKmlGeoFromWkt(final String wkt) throws CatalogTransformerException {
+    if (StringUtils.isBlank(wkt)) {
+      throw new CatalogTransformerException(
+          "WKT was null or empty. Unable to preform KML Transform on Metacard.");
+    }
+
+    com.vividsolutions.jts.geom.Geometry geo = readGeoFromWkt(wkt);
+    Geometry kmlGeo = createKmlGeo(geo);
+    if (!POINT_TYPE.equals(geo.getGeometryType())) {
+      kmlGeo = addPointToKmlGeo(kmlGeo, geo.getCoordinate());
+    }
+    return kmlGeo;
+  }
+
+  private static Geometry createKmlGeo(com.vividsolutions.jts.geom.Geometry geo)
+      throws CatalogTransformerException {
+    Geometry kmlGeo;
+    if (POINT_TYPE.equals(geo.getGeometryType())) {
+      kmlGeo = createPointGeo((Point) geo);
+    } else if (LINES_STRING_TYPE.equals(geo.getGeometryType())) {
+      kmlGeo = createLineStringGeo((LineString) geo);
+    } else if (POLYGON_TYPE.equals(geo.getGeometryType())) {
+      kmlGeo = createPolygonGeo((Polygon) geo);
+    } else if (geo instanceof GeometryCollection) {
+      kmlGeo = createGeometryCollectionGeo(geo);
+    } else {
+      throw new CatalogTransformerException(
+          "Unknown / Unsupported Geometry Type '"
+              + geo.getGeometryType()
+              + "'. Unale to preform KML Transform.");
+    }
+    return kmlGeo;
+  }
+
+  private static Geometry createGeometryCollectionGeo(com.vividsolutions.jts.geom.Geometry geo)
+      throws CatalogTransformerException {
+    Geometry kmlGeo;
+    List<Geometry> geos = new ArrayList<>();
+    for (int xx = 0; xx < geo.getNumGeometries(); xx++) {
+      geos.add(createKmlGeo(geo.getGeometryN(xx)));
+    }
+    kmlGeo = KmlFactory.createMultiGeometry().withGeometry(geos);
+    return kmlGeo;
+  }
+
+  private static Geometry createPolygonGeo(Polygon geo) {
+    Geometry kmlGeo;
+    Polygon jtsPoly = geo;
```

Comment:
❓ Can we not just use `geo` or rename the argument to `jtsPoly` here instead of just copying it?